morepath/tests/test_path_directive.py

```diff
@@ -1802,7 +1800,7 @@ def link(self, request):
     c = Client(App())
 
     response = c.get("/?s=sim~ple")
-    assert response.body == "View: sim~ple".encode("utf-8")
+    assert response.body == b"View: sim~ple"
```

Comment:
This was new to me! Looks good: https://github.com/asottile/pyupgrade/issues/138